Formula/geph4.rb

```diff
@@ -0,0 +1,20 @@
+class Geph4 < Formula
+  desc "Modular Internet censorship circumvention system to deal with national filtering"
+  homepage "https://geph.io/"
+  url "https://github.com/geph-official/geph4/archive/v4.1.2.tar.gz"
+  sha256 "7b2a9a7532cf1780b8b3c7a421ae6def63ad0948570813bd251dd50befb3ff77"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+
+  def install
+    File.delete("Cross.toml")
+    remove_dir(".cargo")
```

Comment:
Why do these need to be removed?

Reply:
Otherwise the build would fail

Reply:
these files are supposed to be used when cross-compiling macOS binaries on Linux, but they can fail the built on macOS